endpoints/_includes/functions.php

```diff
@@ -827,7 +853,8 @@ function getResultsString($filtered_list)
 
 function getServiceBodyCoverage($latitude, $longitude)
 {
-    $search_results = helplineSearch($latitude, $longitude);
+	getBMLTserver($latitude, $longitude, 1);
```

Comment:
spacing issues need to fix to keep consistency with formatting

Reply:
Looking like a possible tab vs spacebar autoformat issue? They align evenly on the editor I used. Will run through another editor to see.